call-home/src/bin/callhome/main.rs

```diff
@@ -33,6 +33,10 @@ struct CliArgs {
     #[arg(short, long, default_value = DEFAULT_NAMESPACE)]
     namespace: String,
 
+    /// Save the report.
+    #[clap(long, short)]
+    save_report: bool,
```

Comment:
Suggest renaming and documenting what it means, example:
```suggestion
    /// Sends the report to the remote collection endpoint.
    #[clap(long, short)]
    sendReport: bool,
```

Reply:
done